### PR TITLE
add 儿戏亲子游 to showcase

### PR DESCRIPTION
**Pull request by nowaits**
add 儿戏亲子游 to showcase, thanks :)

### CODE REVIEW

**Comment by facebook-github-bot-9** [comment]
By analyzing the blame information on this pull request, we identified @brentvatne and @manonthemat to be potential reviewers.

**Comment by facebook-github-bot-9** [comment]
Thank you for your pull request and welcome to our community. We require contributors to sign our Contributor License Agreement, and we don't seem to have you on file. In order for us to review and merge your code, please sign up at https://code.facebook.com/cla - and if you have received this in error or have any questions, please drop us a line at cla@fb.com. Thanks!

**Comment by facebook-github-bot-7** [comment]
Thank you for signing our Contributor License Agreement. We can now accept your code for this (and any) Facebook open source project. Thanks!

File: Libraries/CameraRoll/RCTCameraRollManager.m [hunk 1]
```diff
@@ -205,7 +205,7 @@ static void RCTResolvePromise(RCTPromiseResolveBlock resolve,
           }];
         }
       }];
-    } else {
+    } else if (group == nil) {
```
Comment (javache) [inline Libraries/CameraRoll/RCTCameraRollManager.m]: Please revert this change.

**Comment by nowaits** [comment]
reverted :)
